recipes-core/cvm-reverse-proxy-server/files/cvm-reverse-proxy-server-init

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+### BEGIN INIT INFO
+# Provides:          cvm-reverse-proxy-server
+# Required-Start:    $network $remote_fs $syslog
+# Required-Stop:     $network $remote_fs $syslog
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: Start CVM Reverse Proxy Server at boot time
+# Description:       Enable CVM Reverse Proxy Server service.
+### END INIT INFO
+
+DAEMON=/usr/bin/proxy-server
+NAME=cvm-reverse-proxy-server
+DESC="CVM Reverse Proxy Server"
+DAEMON_ARGS="--listen-addr=https://localhost:7946 --target-addr=localhost:7936 --server-attestation-type azure-tdx"
```

Comment:
is this how it should be executed for the server side?

Reply:
`--target-addr` should include the protocol, ie `http://localhost:7936` in this case, and listen addr only the host and port, ie `localhost:7946`

Reply:
fixed it on both client and server side.
But why is it with protocol for the target and without for the listen addr?

Reply:
It's common for listen addr not to include the protocol. What kind of protocol they bind to is application-defined logic. For the proxy target on the other hand, it's a critical part of the URL.